decisions/wc_class_decisions.txt

```diff
@@ -1876,7 +1876,65 @@ targetted_decisions = {
 	}
 
 	### Hunter ###
+	# Adds tamed beast modifier 
+	tame_beast = {
+		filter = self
+		ai_target_filter = self
+		ai_check_interval = 48
 
+		only_playable = yes
+
+		potential = {
+			is_untouchable_trigger = no
+
+			is_adult = yes
+			is_incapable = no
+			prisoner = no
+
+			NOT = { has_character_flag = tame_beast_ongoing_flag }
+
+			is_hunter_class_trigger = yes
+		}
+
+		allow = {
+			show_only_failed_conditions = yes
+			
+			is_inaccessible_trigger = no
+
+			custom_tooltip = {
+				text = has_not_used_decision_5_years_tooltip
+				hidden_tooltip = {
+					OR = {
+						NOT = { has_character_flag = tame_beast_cooldown_flag }
+						had_character_flag = { flag = tame_beast_cooldown_flag years = 5 }
+					}
+				}
+			}
+
+			has_3_4_hunter_class_or_higher_trigger = yes
```

Comment:
1-4 lvls have `text_effect_tame_beast`, but actually can't use it?

Reply:
No, there's another problem: you need to declare the scripted trigger first.
Info:
https://ck2.paradoxwikis.com/Scripting#Scripted_block

Also, instead of a new trigger, better use a combination of two triggers: `has_3_4_physical_class_or_higher_trigger` and `is_hunter_class_trigger`.